DataHub/Integration/04_ManageChannels.md

```diff
@@ -0,0 +1,132 @@
+
+> [Info] By default, the *Actindo Basic* channel has been predefined in the *Core1 Platform*.
+
+#### Procedure
+
+*DataHub > Settings > Tab CHANNELS*
+
+![Channel list](../../Assets/Screenshots/DataHub/Settings/Channels/ChannelList.png "[Channel list]")
+
+1. Click the channel you want to edit in the list of channels.   
+  The *Edit channel* view is displayed.
+
+  ![Edit channel](../../Assets/Screenshots/DataHub/Settings/Channels/EditChannel.png "[Edit channel]")
+
+2. Edit the desired data of the channel in the corresponding fields.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Edit channel* view is closed.  
+
+
+
+## Deactivate a channel
+
+Deactivate a channel when it should not be used for a certain time.
+The channel is no longer available for offers when it is inactive.
+Further, it is recommended to deactivate a channel instead of deleting it to prevent any problems because of existing connections, see [Deactivate a channel](#deactivate-a-channel).
+
+#### Prerequisites
+
+At least one channel has been created, see [Create a channel](#create-a-channel).
+
+> [Info] By default, the *Actindo Basic* channel has been predefined in the *Core1 Platform*.
+
+#### Procedure
+
+*DataHub > Settings > Tab CHANNELS*
+
+![Channel list](../../Assets/Screenshots/DataHub/Settings/Channels/ChannelList.png "[Channel list]")
+
+1. Click the channel you want to edit in the list of channels.   
+  The *Edit channel* view is displayed.
+
+  ![Edit channel](../../Assets/Screenshots/DataHub/Settings/Channels/EditChannel.png "[Edit channel]")
+
+2. Disable the *Active* toggle.
+
+3. Click the [SAVE] button in the upper right corner.   
+  The channel has been deactivated. The *Edit channel* view is closed.
+
+
+
+## Delete a channel
+
+You can delete a channel if it is no longer needed. As there are usually existing connections via a channel, it is highly recommended not to delete a channel. Instead, you can deactivate a channel and thus prevent its use.
+
+#### Prerequisites
+
+At least one channel has been created, see [Create a channel](#create-a-channel).
+
+> [Info] By default, the *Actindo Basic* channel has been predefined in the *Core1 Platform*. The *Actindo Basic* channel cannot be deleted.
+
+#### Procedure
+
+*DataHub > Settings > Tab CHANNELS*
+
+![Channel list](../../Assets/Screenshots/DataHub/Settings/Channels/ChannelList.png "[Channel list]")
+
+1. Select the checkbox of the channel you want to delete in the list of channels.    
+  The editing toolbar is displayed above the channel list.
+
+2. Click the [DELETE] button in the toolbar.  
+  The channel has been deleted. The deletion cannot be undone.
+
+  > [Info] If the selected channel cannot be deleted, an error message is displayed in the upper right corner. To prevent the channels use, it is recommended to [deactivate the channel](#deactivate-a-channel) instead of delete it.
```

Comment:
```suggestion
  > [Info] If the selected channel cannot be deleted, an error message is displayed in the upper right corner. To prevent the channels use, it is recommended to [deactivate the channel](#deactivate-a-channel) instead of deleting it.
```